src/app/core/title/title.service.ts

```diff
@@ -8,6 +8,7 @@ import { environment as env } from '@env/environment';
 
 @Injectable()
 export class TitleService {
+  private unsubscribe$ = new Subject();
```

Comment:
This isn't doing anything now.

In order to unsubscribe the observables, you will have to emit (and complete) the Subject's stream when the service is destroyed.

```
this.unsubscribe$.next();
this.unsubscribe$.complete();
```